app/middlewares/auth.js

```diff
@@ -0,0 +1,21 @@
+const sessionManager = require('./../services/sessionManager'),
+  User = require('../models').User;
+
+exports.secure = (req, res, next) => {
+  const auth = req.headers[sessionManager.HEADER_NAME];
+
+  if (auth) {
+    const email = sessionManager.decode(auth);
+    User.findOne({ where: email }).then(userAuthenticated => {
+      if (userAuthenticated) {
+        req.email = userAuthenticated;
```

Comment:
you aren't using this anywhere, also you are saving in an attribute named `email` a user, that is a little bit confusing.